### DISCUSSION TITLE
about:blank

### DISCUSSION THREAD

**Discussion by minding-myown-business** [catppuccin/catppuccin#2623] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

The `<body>` of `about:blank` should be the background colour of the user's preferred catppuccin flavour.

### Any additional comments?

`about:blank` is an entirely blank web page that can be accessed by typing `about:blank` in the address bar of all major web browsers.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by Mouadhbendjedidi** [comment]
> As a security precaution­, the browser prohibits extensions from affecting its built-in pages (like chrome://version, the standard new tab page as of Chrome 61, about:addons, and so on) as well as other extensions­' pages. Each browser also restricts access to its own extensions gallery (like Chrome Web Store or AMO).

well, we can't do that with stylus sadly

**Comment by sgoudham** [comment]
Closing as not planned since it's not possible (and I struggle to see the value in such a userstyle)
